tox.ini

```diff
@@ -11,5 +11,5 @@ passenv=ANCHORE_* DOCKER_* NO_PROXY
 commands=
     coverage erase
     coverage run -m pytest --junitxml=.tox/results/tox/junit-{envname}.xml {posargs}
-    coverage report --include=*anchore_engine* --omit=*tests*
-    coverage html --include=*anchore_engine* --omit=*tests*
+    - coverage report --include=*anchore_engine* --omit=*tests*
```

Comment:
there are some occasions where coverage report may be empty. adding the `-` allows tox to exit without error code